prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -428,12 +428,57 @@ ssize_t rxr_pkt_post_ctrl_or_queue(struct rxr_ep *ep, int entry_type, void *x_en
 
 ssize_t rxr_pkt_wait_handshake(struct rxr_ep *ep, fi_addr_t addr, struct rxr_peer *peer)
```

Comment:
Is there any place where we are still using function `rxr_pkt_wait_handshake`? If we just replace `rxr_pkt_wait_handshake` with `rxr_pkt_trigger_handshake`, should we remove this function? If not, the change looks fine to me.

Reply:
The zero-copy patch is still using `rxr_pkt_wait_handshake`

Reply:
okay.